src/webTracer.ts

```diff
@@ -0,0 +1,35 @@
+import { WebTracerProvider } from '@opentelemetry/sdk-trace-web';
+import { SimpleSpanProcessor } from '@opentelemetry/sdk-trace-base';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+import { Resource } from '@opentelemetry/resources';
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+
+import { FetchInstrumentation } from '@opentelemetry/instrumentation-fetch';
+import { LongTaskInstrumentation } from '@opentelemetry/instrumentation-long-task';
+import { UserInteractionInstrumentation } from '@opentelemetry/instrumentation-user-interaction';
+import { DocumentLoadInstrumentation } from '@opentelemetry/instrumentation-document-load';
+
+const WebTracer = async () => {
+
+  const provider = new WebTracerProvider({
+    resource: new Resource({
+      [SemanticResourceAttributes.SERVICE_NAME]: 'kaoto-ui',
+    }),
+  });
+
+  provider.addSpanProcessor(new SimpleSpanProcessor(new OTLPTraceExporter({ headers: {} })));
+  provider.register();
+
+  registerInstrumentations({
+    tracerProvider: provider,
+    instrumentations: [
+      new FetchInstrumentation(),
+      new DocumentLoadInstrumentation(),
+      new LongTaskInstrumentation(),
+      new UserInteractionInstrumentation()
+    ],
+  });
+};
+
+export default WebTracer;
```

Comment:
@helio-frota tiny change but do you mind to add a new line here? sorry I know it's nitpicking 😅

Reply:
sure done!